src/components/NavBar.tsx

```diff
@@ -6,17 +6,18 @@ import '../styles/NavBar.scss';
 
 export function NavBar() {
     const [closeBurger, setBurgerState] = useState(true);
+    const [currentPage, setCurrentPage] = useState('Home');
 
     return (
         <div className='header'>
             <div className={closeBurger ? 'nav-bar' : 'nav-bar-mobile'} data-testid="nav-bar">
-                <Link to='/' className = 'nav-link'> Home </Link>
-                <Link to='/spirit' className = 'nav-link'>Spirit </Link>
-                <Link to='/opportunity'className = 'nav-link'>Opportunity</Link>
-                <Link to='/curiosity'className = 'nav-link'>Curiosity</Link>
-                <Link to='/timeline'className = 'nav-link' > Timeline</Link>
+                <Link to='/' className = 'nav-link' onClick={()=>{setBurgerState(true); setCurrentPage('Home')}}> Home </Link>
```

Comment:
You could make the onclick to have a function instead of the repeated code here, maybe a 
```
function nameOfFunction(pageName: string) {
    setBurgerState(true);
    setCurrentPage(pageName);
}
```